src/main/pages/che-7/administration-guide/proc_developing-grafana-dashboards.adoc

```diff
@@ -0,0 +1,32 @@
+[id="developing-grafana-dashboards_{context}"]
+= Developing Grafana dashboards
+
+// todo rolout
+
+Grafana offers the possibility to add custom panels.
+
+
+[discrete]
+== Procedure
+
+To add a custom panel, use the *New dashboard* view.
+
+. The first section is for defining *Queries to* using the _Prometheus Query Language_ to construct a specific metric, as well as to modify it with various aggregation operators.
++
+.New Grafana dashboard: Queries to
+image::monitoring/monitoring-che-new-grafana-dashboard-queries.png[link="{imagesdir}/monitoring/monitoring-che-new-grafana-dashboard-queries.png"]
+
+. The next section is *Visualisation* where you choose a metric to be shown in the following visual in the form of a graph, gauge, heatmap, or others.
++
+.New Grafana dashboard: Visualization
+image::monitoring/monitoring-che-new-grafana-dashboard-visualization.png[link="{imagesdir}/monitoring/monitoring-che-new-grafana-dashboard-visualization.png"]
+
+To save changes to the dashboard, click the btn:[Save] button, and copy and paste the JSON code to the deployment.
+
```

Comment:
To load changes in configurations for a running Grafana deployment, `oc process/oc apply` commands can be used.
First, run `oc process/oc delete` for che-monitoring.yaml to remove it, in case it was deployed via "oc new-app" command)
`oc process -f che-monitoring.yaml | oc delete  -f -`

Then use the following command to redeploy your Grafana with new configuration:
`oc process -f che-monitoring.yaml | oc apply  -f - | oc rollout latest grafana`

Reply:
Thanks, @mkuznyetsov. I added it.